spice_agent/inference/actions.py

```diff
@@ -103,6 +106,37 @@ def _update_inference_job(
             else:
                 raise exception
 
+    def _get_stable_diffusion_options(self, options: Dict[str, Any]) -> dict:
+        """
+        Parses any inference options that may be defined for
+        StableDiffusionPipeline
+        """
+
+        stable_diffusion_options: dict = {}
+
+        if "negative_prompt" in options:
+            stable_diffusion_options["negative_prompt"] = options["negative_prompt"]
+
+        if "guidance_scale" in options:
+            stable_diffusion_options["guidance_scale"] = options["guidance_scale"]
+
+        if "num_inference_steps" in options:
+            stable_diffusion_options["num_inference_steps"] = options[
+                "num_inference_steps"
+            ]
+
+        if "seed" in options:
+            # If seed is -1, we generate a random seed and update
+            # the inference job.
+            # Note, completely reproducible results are not guaranteed across
+            # PyTorch releases.
+            if options["seed"] == -1:
+                options["seed"] = torch.seed()
+
+            stable_diffusion_options["generator"] = torch.manual_seed(options["seed"])
```

Comment:
need to have this seed come from the backend
```suggestion
            stable_diffusion_options["generator"] = torch.manual_seed(options["seed"])
```